app/src/main/res/layout-port/session_layout.xml

```diff
@@ -24,22 +24,18 @@
         <ImageView
             android:id="@+id/session_no_video_view"
             android:layout_width="wrap_content"
-            android:layout_height="13sp"
-            android:layout_marginLeft="3dp"
+            android:layout_height="22sp"
             android:adjustViewBounds="true"
-            android:baseline="12sp"
-            android:scaleType="fitEnd"
+            android:scaleType="fitStart"
             android:src="@drawable/ic_novideo" />
 
         <ImageView
             android:id="@+id/session_bell_view"
             android:layout_width="wrap_content"
-            android:layout_height="13sp"
-            android:layout_marginLeft="3dp"
+            android:layout_height="22sp"
```

Comment:
The bell icon adjusts to the height (without the red bar) of the camera icon next to it by adding `android:padding="3dp"` here. I think this looks a bit better.

:heavy_check_mark: Tested on:

- Pixel 6, Android 13, portrait and landscape modes
- 10" tablet, Android 13 in landscape and landscape split mode.